kratos/processes/from_json_check_result_process.h

```diff
@@ -0,0 +1,921 @@
+    {
+        const auto it = find(rVariable.Key());
+        if (it != end()) {
+            return it->second;
+        } else {
+            KRATOS_ERROR << "Not allocated Variable: " << rVariable.Name() << std::endl;
+        }
+    }
+
+    /**
+     * @brief This method retrieves the variable database
+     * @param rVariable The variable to be retrieved
+     * @tparam TVariableType The variable type considered
+     */
+    template<class TVariableType>
+    const VariableDatabase& GetVariableData(const TVariableType& rVariable) const
+    {
+        const auto it = find(rVariable.Key());
+        if (it != end()) {
+            return it->second;
+        } else {
+            KRATOS_ERROR << "Not allocated Variable: " << rVariable.Name() << std::endl;
+        }
+    }
+
+//     /**
+//      * @brief This method retrieves the interpolated value from the database
+//      * @param rVariable The variable to be retrieved
+//      * @param EntityIndex The index of the entity (not the entity Id, the index in the database)
+//      * @param Time The time value to be retrieved
+//      * @param ComponentIndex The component index of the vector/array
+//      * @param GPIndex The Gauss point index
+//      * @tparam TVariableType The variable type considered
+//      */
+//     template<class TVariableType>
+//     const double GetValue(
+//         const TVariableType& rVariable,
+//         const IndexType EntityIndex,
+//         const double Time,
+//         const SizeType ComponentIndex = 0,
+//         const SizeType GPIndex = 0
+//         ) const
+//     {
+//         return GetVariableData(rVariable).GetValue(EntityIndex, Time, ComponentIndex, GPIndex);
+//     }
+
+    /**
+     * @brief This method set the values into the tables
+     * @param rValuesX The values of the X axis
+     * @param rValuesY The values of the Y axis
+     * @param rVariable The variable to be retrieved
+     * @param EntityIndex The index of the entity (not the entity Id, the index in the database)
+     * @param ComponentIndex The component index of the vector/array
+     * @param GPIndex The GP index
+     * @tparam TVariableType The variable type considered
+     */
+    template<class TVariableType>
+    void SetValues(
+        const Vector& rValuesX,
+        const Vector& rValuesY,
+        const TVariableType& rVariable,
+        const IndexType EntityIndex,
+        const SizeType ComponentIndex = 0,
+        const SizeType GPIndex = 0
+        )
+    {
+        auto it = find(rVariable.Key());
+        if (it != end()) {
+            auto& r_database = it->second;
+            r_database.SetValues(rValuesX, rValuesY, EntityIndex, ComponentIndex, GPIndex);
+        } else {
+            KRATOS_ERROR << "Not allocated Variable: " << rVariable.Name() << std::endl;
+        }
+    }
+
+    /**
+     * @brief This method clears the database of the common column vector
+     */
+    void ClearCommonColumn()
+    {
+        mCommonColumn.clear();
+    }
```

Comment:
I didn't found another use of this method. If I'm not wrong, I'd remove the function and do `mCommonColumn.clear()` in the `Clear` method.